{{cookiecutter.repo_name}}/.gitignore

```diff
@@ -0,0 +1,86 @@
+*.py[cod]
+
+*.log
+
+# C extensions
+*.so
+
+# Packages
+*.egg
+*.egg-info
+dist
+build
+eggs
+parts
+bin
+var
+sdist
+develop-eggs
+.installed.cfg
+lib
+lib64
+__pycache__
+
+# Installer logs
+pip-log.txt
+
+# Django
+default.db
+
+# Static assets
+media/cache/
+assets/
+node_modules/
+{{cookiecutter.repo_name}}/static/bower_components/
+{{cookiecutter.repo_name}}/static/build/
+npm-debug.log
+
+# Unit test / coverage reports
+coverage/
```

Comment:
That was the JS coverage directory. Static assets will come in a future iteration.